pages/Elevator-Plate.md

```diff
@@ -0,0 +1,7 @@
+## Obtaining
+The Elevator Plate can be crafted in an [Enchanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+Elevator Plates are automatically connected when sharing the X and Z coordinates.
+
+Upon stepping on an Elevator Plate, the user is prompted with a book GUI of all of the storeys. By clicking on a story, the user can teleport to it.
```

Comment:
*stories, although it's called "floors" in Slimefun so you should keep this consistent.

Reply:
Idk about floors but storeys and stories are both correct. `storeys` is from British English and `stories` is from American English.

Reply:
huh okay then. I think it should still be consistent with the ingame name then though